main.py

```diff
@@ -13,27 +14,55 @@ def tweet(message: str):
     auth.set_access_token(os.getenv("ACCESS_TOKEN"), os.getenv("ACCESS_TOKEN_SECRET"))
 
     api = tweepy.API(auth)
-    api.update_status(message)
+    try:
+        api.update_status(message)
+        logger.info("The gazette was posted on twitter!")
+    except tweepy.TweepError as e:
+        logger.exception(e)
+
+
+def create_maria_quiteria_api_token():
+    headers = {"content_Type": "application/json"}
+    url = f"{os.getenv('MARIA_QUITERIA_API_HOST')}/token/"
+    data = {
+        "username": f"{os.getenv('MARIA_QUITERIA_USERNAME_CREDENCIALS')}",
+        "password": f"{os.getenv('MARIA_QUITERIA_PASSWORD_CREDENCIALS')}",
+    }
+
+    token_response = requests.post(url, headers=headers, data=data)
+    logger.info("Getting token from Maria Quitéria")
+
+    if token_response.status_code == 200:
+        return f"Bearer {token_response.json()['access']}"
+    else:
+        logger.error(token_response.raise_for_status())
```

Comment:
```suggestion
logger.info("Getting token from Maria Quitéria")
token_response = requests.post(url, headers=headers, data=data)
token_response.raise_for_status()
return token_response.json()['access']
```

O “raise for status” vai levantar uma exceção que seja diferente de status Code de sucesso. Por isso não precisamos do if pq ele já vai fazer isso pra gente.
O formato do Token deve ser montado na criação do Header. Assim fica mais claro lá e aqui mantemos o método com uma só responsabilidade: criar o Token. Faz sentido?